mollie/api/objects/order_line.py

```diff
@@ -0,0 +1,141 @@
+from .base import Base
+
+
+class OrderLine(Base):
+
+    STATUS_CREATED = 'created'
+    STATUS_AUTHORIZED = 'authorized'
+    STATUS_PAID = 'paid'
+    STATUS_SHIPPING = 'shipping'
+    STATUS_CANCELED = 'canceled'
+    STATUS_REFUNDED = 'refunded'
+    STATUS_COMPLETED = 'completed'
+
+    @classmethod
+    def get_object_name(cls):
+        return 'lines'
+
+    @property
+    def id(self):
+        return self._get_property('id')
+
+    @property
+    def resource(self):
+        return self._get_property('resource')
+
+    @property
+    def order_id(self):
+        return self._get_property('orderId')
+
+    @property
+    def type(self):
+        return self._get_property('type')
+
+    @property
+    def name(self):
+        return self._get_property('name')
+
+    @property
+    def status(self):
+        return self._get_property('status')
+
+    @property
+    def is_cancelable(self):
+        return self._get_property('isCancelable')
+
+    @property
+    def quantity(self):
+        return self._get_property('quantity')
+
+    @property
+    def quantity_shipped(self):
+        return self._get_property('quantityShipped')
+
+    @property
+    def amount_shipped(self):
+        return self._get_property('amountShipped')
+
+    @property
+    def quantity_refunded(self):
+        return self._get_property('quantityRefunded')
+
+    @property
+    def amount_refunded(self):
+        return self._get_property('amountRefunded')
+
+    @property
+    def quantity_canceled(self):
+        return self._get_property('quantityCanceled')
+
+    @property
+    def amount_canceled(self):
+        return self._get_property('amountCanceled')
+
+    @property
+    def unit_price(self):
+        return self._get_property('unitPrice')
+
+    @property
+    def discount_amount(self):
+        return self._get_property('discountAmount')
+
+    @property
+    def total_amount(self):
+        return self._get_property('totalAmount')
+
+    @property
+    def vat_rate(self):
+        return self._get_property('vatRate')
+
+    @property
+    def vat_amount(self):
+        return self._get_property('vatAmount')
+
+    @property
+    def sku(self):
+        return self._get_property('sku')
+
+    @property
+    def image_url(self):
+        return self._get_property('imageUrl')
+
+    @property
+    def product_url(self):
+        return self._get_property('productUrl')
+
+    @property
+    def created_at(self):
+        return self._get_property('createdAt')
+
+    def cancel(self):
+        """Cancel order line.
+
+        Deleting an order line causes the order line status to change to canceled.
+        """
+        # Import OrderLines locally to avoid circular import
+        from ..resources.order_lines import OrderLines
+
+        return OrderLines(self.client).with_parent_id(self.order_id).delete(self.id)
```

Comment:
The documentation code examples aren't matching the description and your response. Other than that, it's a weird deviation from the REST standard (and the rest of the API). What's the reason for this change?

Reply:
You're totally right, our bad! 😕 

The reason for the change is to allow users of the API to cancel multiple lines in one API call, instead of requiring them to do multiple API calls.

We thought about combining the `cancelOrder` and `cancelOrderlines` API calls, but since `cancelOrder` really indicates canceling the whole order (rather than some or all lines), we decided to keep the separate endpoint.

Reply:
The documentation states it's a DELETE request, i guess your reference to POST from above is a mistake?

Reply:
Ah, yes..